docs/user/dql/functions.rst

```diff
@@ -1383,9 +1420,189 @@ NOW
 Description
 >>>>>>>>>>>
 
-Specifications:
+Returns the current date and time as a value in 'YYYY-MM-DD hh:mm:ss.nnnnnn' format. The value is expressed in the cluster time zone.
+`NOW()` returns a constant time that indicates the time at which the statement began to execute. This differs from the behavior for `SYSDATE() <#sysdate>`_, which returns the exact time at which it executes.
+
+Return type: DATETIME
+
+Specification: NOW() -> DATETIME
+
+Example::
+
+    > SELECT NOW() as value_1, NOW() as value_2;
+    fetched rows / total rows = 1/1
+    +----------------------------+----------------------------+
+    | value_1                    | value_2                    |
+    |----------------------------+----------------------------|
+    | 2022-08-02 15:39:05.173069 | 2022-08-02 15:39:05.173069 |
+    +----------------------------+----------------------------+
```

Comment:
I get this result from mySQL for this query. Should `NOW` return with `.nnnnnn`?
```
mysql> SELECT NOW() as value_1, NOW() as value_2;
+---------------------+---------------------+
| value_1             | value_2             |
+---------------------+---------------------+
| 2022-09-29 07:30:55 | 2022-09-29 07:30:55 |
+---------------------+---------------------+
1 row in set (0.00 sec) 
```